cfme/tests/automate/test_method.py

```diff
@@ -264,3 +265,90 @@ def test_task_id_for_method_automation_log(request, generic_catalog_item):
     wait_for(lambda: service_request.request_state == "active", fail_func=service_request.reload,
              timeout=60, delay=3)
     result.validate_logs()
+
+
+@pytest.mark.meta(blockers=[BZ(1704439)])
+@pytest.mark.meta(server_roles="+notifier")
+def test_send_email_method(request, klass):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/20h
+        caseimportance: high
```

Comment:
No need of mentioning the `caseimportance` if it is high and `testtype` if it is functional.